src/include/update/rootpart.rb

```diff
@@ -360,7 +360,14 @@ def RootPartitionDialog(flavor)
             selected,
             freshman
           )
-          if Ops.get_string(freshman, :name, "unknown") == "unknown"
+
+          if freshman[:fs] == :reiserfs
+            cont = false
+            Report.Error(_("Reiserfs is not supported anymore.\n" \
+                            "Please migrate your data to other " \
+                            "filesystem before perform the upgrade.\n\n" \
+                            "More details can be found in the release notes."))
```

Comment:
Maybe we should not mention release notes because, for future releases, that information will not be on release notes anymore (and we would need to update this string accordingly again).